src/main/java/cuchaz/enigma/bytecode/translators/LocalVariableFixVisitor.java

```diff
@@ -31,50 +34,91 @@ public MethodVisitor visitMethod(int access, String name, String descriptor, Str
 
 	private class Method extends MethodVisitor {
 		private final MethodDefEntry methodEntry;
-		private boolean hasLvt;
+		private final Map<Integer, String> parameterNames = new HashMap<>();
+		private final Map<Integer, Integer> parameterIndices = new HashMap<>();
+		private boolean hasParameterTable;
+		private int parameterIndex = 0;
 
 		Method(int api, MethodDefEntry methodEntry, MethodVisitor visitor) {
 			super(api, visitor);
 			this.methodEntry = methodEntry;
+
+			int lvtIndex = methodEntry.getAccess().isStatic() ? 0 : 1;
```

Comment:
`lvIndex` should be more accurate because these are not necessarily in a local variable table